### ISSUE TITLE
@charset should be excluded from quotes rule

### ISSUE THREAD

**Issue by natewiebe13** [sasstools/sass-lint#1004]
What version of Sass Lint are you using? 1.10.2

Please include any relevant parts of your configuration
```yml
rules:
  quotes:
    - 2
    -
      style: single
```

What did you do? Please include the actual source code causing the issue.
```css
@charset "UTF-8";
```

What did you expect to happen?
The linter should identify any place that double quotes are being used, but exclude `@charset` since the value must use double quotes. Source: https://developer.mozilla.org/en/docs/Web/CSS/@charset

What actually happened? Please include any error messages given to you by Sass Lint.
```
public/css/styles.scss
  1:10  error  Strings must use single quotes  quotes
```

**Comment by S0AndS0** [comment]
The following isn't a _fix_ per-say but more of some ways to disable lint checks.


___


Using <kbd>Ctrl</kbd> <kbd>f</kbd> on the [example config/converter](https://sasstools.github.io/make-sass-lint-config/) showed...


```yaml
rules:
  quotes:
    - 1
    - style: single
```

... and by changing the **`1`** to a **`0`** I was able to silence the warnings ...


```yaml
rules:
  quotes:
    - 0
    - style: single
```

Example of checking if quote warnings are gone with Bash shell...

```Bash
npm run sass-lint | grep -q -- 'quotes'
if [ "$?" -gt '0' ]; then
  printf 'Quote warnings are persisting!\n' >&2
else
  printf 'Groovy, no more quote warnings!\n'
fi
```

... which _should_ report something like the following if the configuration file is getting read...


```Bash
#> Groovy, no more quote warnings!
```


However, this silences all warnings so far and not just for _`@charset`_, so for disabling linting for spicific things within the source I think that the main [`ReadMe.md`](https://github.com/sasstools/sass-lint#disable-a-rule-for-the-entire-file) file has more than one section on the topic.
